quoridor-logic-server/src/main/java/com/rl/mpquoridor/models/common/WebSocketMessage.java

```diff
@@ -0,0 +1,9 @@
+package com.rl.mpquoridor.models.common;
+
+import lombok.Data;
+
+@Data
+public class WebSocketMessage {
+    private String type;
+    private String gameID;
```

Comment:
uuid?

Reply:
no, thids object going to be parsed to json
UUID is object which will be used to make the id